KaktusWatch/Core/Mailing.cs

```diff
@@ -44,7 +37,7 @@ public static IEnumerable<int> GetPromotionTimeFrame(string message)
             foreach (string word in message.Split(' '))
             {
                 int hour;
-                if (Int32.TryParse(word.Replace(".", ""), out hour))
```

Comment:
AFAIK Microsoft guidelines recommend using full type name instead of keywords when using static members.